kong-redis-cluster-1.4.0-0.rockspec

```diff
@@ -1,8 +1,8 @@
 package = "kong-redis-cluster"
-version = "1.3.0-1"
+version = "1.4.0-0"
 source = {
     url = "git://github.com/Kong/resty-redis-cluster",
-    tag = "1.3.0-1"
```

Comment:
The `-1` at the end should not be changed, it is the rockspec revision, it normally starts at 1. Mood anyway, as this has to be removed from this commit.

It is only presenyt in this tag here, because we had to release a rockspec change  in the past. Upon the next version bump the tag should be brought back to simply "x.y.z" without the revision.